wallet/chain/p/backend_visitor.go

```diff
@@ -70,6 +70,11 @@ func (b *backendVisitor) TransferSubnetOwnershipTx(tx *txs.TransferSubnetOwnersh
 	return b.baseTx(&tx.BaseTx)
 }
 
+func (b *backendVisitor) ConvertSubnetTx(tx *txs.ConvertSubnetTx) error {
+	// TODO: handle subnet conversion
```

Comment:
I think I'm happy with leaving this TODO for now- it's unclear how the wallet should behave with a Subnet manager since it's expected for the users to aggregate the warp signature. I'd like to defer to a future PR as we build out more of the new txs

Reply:
I removed this TODO, I misunderstood `backend`. This should be updating the backend if the signers change for the Subnet. The `Owner` key can still be used if the subnet has no validators so we shouldn't be disabling it fully here